tools/GitHubTools.psm1

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+function CloneRepo
+{
+    param(
+        [Parameter(Mandatory)]
+        [string]
+        $OriginRemote,
+
+        [Parameter(Mandatory)]
+        [string]
+        $Destination,
```

Comment:
maybe this could just have a default of `""`

Reply:
`""` would just error I think.

I tried to get a sensible default here, which would be the repo name, but it turns out [that can be complicated](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt-ltdirectorygt)

Reply:
So I decided to leave it as mandatory for now, let the human figure it out

Reply:
https://stackoverflow.com/questions/13839879/how-to-determine-the-humanish-part-of-a-git-repository

Looking into this

Reply:
Ok I have a thing that works I think